arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -201,10 +204,16 @@ public void start(Archive<?> archive) throws Exception {
         // check for "org.wildfly.swarm.allDependencies" flag
         // see DependenciesContainer#addAllDependencies()
         if (archive instanceof DependenciesContainer) {
-            DependenciesContainer depContainer = (DependenciesContainer) archive;
-            if (depContainer.hasMarker("org.wildfly.swarm.allDependencies")) {
+            DependenciesContainer<?> depContainer =  (DependenciesContainer<?>) archive;
+            if (depContainer.hasMarker(DependenciesContainer.ALL_DEPENDENCIES_MARKER)) {
                 munge(depContainer, declaredDependencies);
             }
+        } else if (archive instanceof WebArchive) {
```

Comment:
Nuts, missed that aspect.

I hate how there are essentially two WAR Archive types, one we've done and one in SW.

Someday we should fix that